anchor/src/lib.rs

```diff
@@ -1,13 +1,21 @@
 // SPDX-FileCopyrightText: 2021 Chorus One AG
 // SPDX-License-Identifier: GPL-3.0
 
+use solana_program::pubkey::Pubkey;
+
 #[cfg(not(feature = "no-entrypoint"))]
 pub mod entrypoint;
 mod instruction;
 mod logic;
-mod processor;
-mod state;
-mod token;
+pub mod processor;
+pub mod state;
+pub mod token;
 
-/// Mint authority, mints StSol.
+/// Mint authority, mints bSOL.
 pub const ANCHOR_MINT_AUTHORITY: &[u8] = b"mint_authority";
+
+/// Return the address at which the Anchor instance should live that belongs to
+/// the given Solido instance.
+pub fn find_instance_address(anker_program_id: &Pubkey, solido_instance: &Pubkey) -> (Pubkey, u8) {
```

Comment:
Does that mean that Solido has to create the Anker instance? I think this is not needed. They should be two independent programs.